doc/gtsam.lyx

```diff
@@ -2191,8 +2203,8 @@ reference "fig:example"
 
 , along with covariance ellipses shown in green.
  These covariance ellipses in 2D indicate the marginal over position, over
- all possible orientations, and show the area which contain 68.26% of the
- probability mass (in 1D this would correspond to one standard deviation).
+ all possible orientations, and show the area which contain 99.9996% of the
+ probability mass.
```

Comment:
`These 5-sigma covariance ellipses in 2D indicate the marginal over position, over
 all possible orientations, and show the areas which contain 99.9996% of the
 probability mass.`